lib/maropost/contact.rb

```diff
@@ -35,14 +35,14 @@ def initialize(opts = {})
       self.email = data[:email]
       self.phone_number = data[:phone]
       self.cell_phone_number = data[:cell_phone_number]
-      self.do_not_contact = data.fetch(:do_not_contact, false)
+      self.allow_emails = data.fetch(:allow_emails) { !DoNotMailList.exists?(self) }
       self.errors = []
       self.lists = {}
       initialize_lists(data)
     end
 
     def allow_emails?
-      @allow_emails ||= email.present? && !do_not_contact && !DoNotMailList.exists?(self)
+      allow_emails
```

Comment:
you don't get this method for free now right?

Reply:
I could change the callers to reference `allow_emails` directly. This is just a convenience method.

Reply:
ok, leave it. I just wanted to make sure there wasn't a helper giving you allow_emails? for free